src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/testcontainers/LocalStackContainer.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.testcontainers
+
+import org.slf4j.LoggerFactory
+import org.springframework.test.context.DynamicPropertyRegistry
+import org.testcontainers.containers.localstack.LocalStackContainer
+import org.testcontainers.containers.output.Slf4jLogConsumer
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.utility.DockerImageName
+import java.io.IOException
+import java.net.ServerSocket
+
+object LocalStackContainer {
```

Comment:
I know you copy and pasted this class, but its not up to our usual high standards! 😁 🤣 
For example the `instance` method returns `null` if localstack is already running!
And I wonder if `log` could/should be the kotlin logger we use everywhere else?